README.md

```diff
@@ -173,7 +193,11 @@ Yank all you want and then do:
 ```vim
 :Telescope neoclip
 ```
-which will show you a history of the yanks that happened in the current session.
+if using `telescope` or
```

Comment:
```suggestion
if using `fzf-lua` or
```

Reply:
Thanks for checking this @kradalby! Reading the current sentence I think it's correct since it refers to the previous code block. Does that make sense or do you find it confusing?

Reply:
Oh you are absolutely right, sorry, I think my brain automatically expects:

If you do this:
	execute this

If you do that:
	execute that

So I think I read it too fast (or didnt read properly) and got confused.